src/Administration/Resources/app/administration/src/module/sw-bulk-edit/component/sw-bulk-edit-save-modal-success/index.js

```diff
@@ -178,7 +179,8 @@ export default {
                         return;
                     }
 
-                    const filename = response.headers['content-disposition'].split('filename=')[1];
+                    const contentDisposition = response.headers['content-disposition'];
+                    const filename = fileReaderUtils.getFilenameFromResponse(contentDisposition);
```

Comment:
Small nitpick: the function name is `getFilenameFromResponse` but it takes in just a string of a header. I would suggest changing the signature to take in the full response, then you don't have to look up which header to pass into this function and it's much easier to use